src/api/app/views/webui/packages/build_reason/_breadcrumb_items.html.haml

```diff
@@ -2,6 +2,6 @@
 %li.breadcrumb-item
   = link_to @package, package_show_path(@project, @package)
 %li.breadcrumb-item
-  = link_to 'Repositoy state', package_binaries_path(@project, @package, @repository.name)
+  = link_to 'Repositoy state', project_package_binaries_path(@project, @package, @repository.name)
```

Comment:
Everywhere else you use `project_name: @project`...

Reply:
Also "repository" is not part of the routes and it should be `repository_name` no? :)

Reply:
ok

Reply:
Still it should be part of the route. Now we end up with

https://obs-reviewlab.opensuse.org/hellcp-work-binary-show/projects/home:Admin:hans/packages/ctris/binaries?repository=15.4